js/modules/k6/html/element_test.go

```diff
@@ -210,8 +210,8 @@ func TestElement(t *testing.T) {
 
 			assert.True(t, ok)
 			assert.Equal(t, 9, len(nodes))
-			assert.Contains(t, nodes[0].Export().(Element).TextContent(), "innerfirst")
-			assert.Contains(t, nodes[8].Export().(Element).TextContent(), "innerlast")
+			assert.Contains(t, nodes[0].Export().(Element).TextContent(), "innerfirst") //nolint:forcetypeassert
```

Comment:
Could make sense to disable `forcetypeassert` linter for all the test files? While I completely see the value of that linter for production Go code, I don't fully see it for tests (as in the worst case, the test would fail?).

I've seen many annotations (+20) like this and most of them are in test files, and I started to feel like ol' ~good~ times in Java with annotations 😅

Reply:
I am not really against this, and it might be easier to do it :shrug: 

cc @grafana/k6-core anyone against it?

Reply:
I agree with Joan, in the context of tests, I always assume the person writing these kind of checks know what they're doing and why, so 👍🏻 on my side 🙇🏻

Reply:
https://github.com/grafana/k6/pull/3704